src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1440,6 +1497,8 @@ void BlockchainLMDB::reset()
     throw0(DB_ERROR(lmdb_error("Failed to drop m_hf_versions: ", result).c_str()));
   if (auto result = mdb_drop(txn, m_properties, 0))
     throw0(DB_ERROR(lmdb_error("Failed to drop m_properties: ", result).c_str()));
+  if (auto result = mdb_drop(txn, m_rct_distribution, 0))
+    throw0(DB_ERROR(lmdb_error("Failed to drop m_properties: ", result).c_str()));
```

Comment:
Maybe `m_rct_distribution_offset = 0;` also should be done here? Just guessing from looking at the content of the ctor.